### DISCUSSION TITLE
Cannot verticaly center notification, throws bug

### DISCUSSION THREAD

**Discussion by rikiless** [filamentphp/filament#5713] [General]
According to [docs](https://filamentphp.com/docs/2.x/admin/appearance#notification-position) I can set `layouts.notifications.vertical_alignment` (small typo here, `layout.` not `layouts.`) to any value of `top`, `center` or `bottom`.

I got this error `UnhandledMatchError Unhandled match case 'center'` when set following in `config/filament.php`:

```php
 'notifications' => [
            'vertical_alignment' => 'center',
// ...
```

Is it bug?

v2.17.2

**Comment by pxlrbt** [comment]
Sounds like a bug, and not sure why this was removed:

@danharrin You removed that option with a "wip" commit in v2 in July 2022: https://github.com/filamentphp/filament/commit/08b73793bcc9ad76cd03f839b677e977665eac73

**Reply by danharrin** [reply]
That commit was part of #3091 that introduced the notifications package, so this isn't a breaking change, the docs were just wrong

**Reply by pxlrbt** [reply]
It is, since we had it as an option in the admin panel before the new notifications package was introduced.

![image](https://user-images.githubusercontent.com/22632550/219680915-0862ac6e-4221-4516-9ff1-7971f31f6c60.png)

But since nobody complained yet, I guess nobody's using it.

**Reply by danharrin** [reply]
oh right, gotcha

**Comment by zepfietje** [comment]
Vertical alignment only supports top and bottom as far as I know.

**Reply by pxlrbt** [reply]
It used to support center and was removed in v2.

**Reply by zepfietje** [reply]
Hmm okay. Let's just update the docs I guess?

**Reply by pxlrbt** [reply]
Not sure. Seems like nobody used it before, because there were no reports and it's gone for over half a year now. 
But it's a breaking changed introduced in v2

**Reply by danharrin** [reply]
Lets remove from the docs, central alignment sounds weird for notifications

**Comment by rikiless** [comment]
Thank you for answers. I don't use GitHub discussions too much. Can the topic be closed or should it leave alone?

**Reply by zepfietje** [reply]
Discussions can't be closed, only marked as answered when it was a question. This discussion can just be left like it is.
